mona_sdk/client.py

```diff
@@ -1054,6 +1057,41 @@ def create_openai_context_class(self, context_class, openai_api_type):
             else get_dict_result(True, app_server_response["response_data"], None)
         )
 
+    @Decorators.refresh_token_if_needed
+    def initiate_csv_upload_request(
+        self,
+        file_path,
+        context_class,
+        context_id_field=None,
+        export_timestamp_field=None,
+    ):
+        """
+        A wrapper function for initiate_csv_upload_request REST endpoint.
+        """
+
+        def custom_bad_response_handler(json_response, status_code):
+            error_message = None
+            if status_code == CLIENT_ERROR_RESPONSE_STATUS_CODE:
```

Comment:
Nit: This whole block can be a bit more organized with a specific function for this firs if and the then just a one line ternary operator

Reply:
I still want to check for both status codes, there might be other cases... how can this be a one-line ternary? Any way extracted the client error to a function

Reply:
OK. Then if,elif,else is the right way to go.